src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +614,119 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
 ```bash
 Successfully updated the mesh with the id: MESH_ID
 ```
+
+## aio api-mesh log-list
+
+The `log-list` command lists the last 15 events for your mesh by rayID.
+
+### Usage
+
+```bash
+aio api-mesh log-list
+```
+
+### Flags
+
+The following arguments are required:
+
+`--filename` allows you to download a YAML or JSON list of rayIDs with the specified filename.
+
+The following arguments are optional.
+
+`-i` or `--ignoreCache` ignores the cached organization, project, and workspace, which allows you to create a mesh in a different workspace. You can also manually [modify the cache](../basic/work-with-mesh.md#projects-and-workspaces).
+
+`--json` outputs the logs in JSON format.
```

Comment:
I do not know. @dthampy ?

Reply:
I believe that when you include this flag, it  instructs the command to output results in JSON format rather than the default table or text format. I have asked Christopher for confirmation on his CLI PR

Reply:
I removed this flag in the CLI now, I took reference from another command and forgot to remove this flag.